irc/config.go

```diff
@@ -4,12 +4,32 @@ import (
 	"code.google.com/p/gcfg"
 	"errors"
 	"log"
+	"crypto/tls"
 )
 
 type PassConfig struct {
 	Password string
 }
 
+// SSLListenConfig defines configuration options for listening on SSL
+type SSLListenConfig struct {
+	SSLCert string
+	SSLKey  string
+}
+
+// Certificate returns the SSL certificate assicated with this SSLListenConfig
+func (conf *SSLListenConfig) Config() (*tls.Config, error) {
+	cert, err := tls.LoadX509KeyPair(conf.SSLCert, conf.SSLKey)
+	if err != nil {
+		log.Fatal("sslconf+sslkey: invalid pair. Error: ", err)
```

Comment:
I think this `log.Fatal` should be removed, since the `error` is returned, and logged with `log.Fatal` in `*Conig.SSLListeners()`